appcues/src/main/java/com/appcues/trait/appcues/TargetElementTrait.kt

```diff
@@ -114,6 +115,7 @@ internal class TargetElementTrait(
         throw AppcuesTraitException(
             message = "multiple non-distinct views (${weightedViews.count()}) matched selector ${selector.toMap()}",
             retryMilliseconds = retryMilliseconds,
+            recoverable = true,
```

Comment:
then on TargetElementTrait we mark as recoverable, so not all Exceptions can be recovered